duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -236,6 +239,11 @@ public void addParam(String name, String value) {
     params.put(name, value);
   }
 
+  public void addAdditionalHeader(String name, String value) throws IllegalArgumentException{
+    validateAdditionalHeader(name, value);
```

Comment:
This is different compared to python client.
In python client we do the validation during canon part. 
But since the validation method would raise exception, I think have the validation during build process make it little bit more backward compatible. If someone don't use this function, they don't have to be aware of this exception.

Reply:
I like this better anyway.  I think it's better to reject an invalid header as early as possible.